package.json

```diff
@@ -19,8 +19,8 @@
     "build:flow": "flow-copy-source src dist/es -i \"**/*.spec.js\"",
     "build": "npm run build:clean && npm run build:css && npm run build:flow && npm run build:rollup",
     "start": "npm run js:watch",
-    "start-demo": "webpack-dev-server --config ./webpack/webpack.config.demo.js",
```

Comment:
Consolidated webpack config below (major simplifications, more sensible defaults in Wepback 4).